src/backend/tools.ts

```diff
@@ -429,7 +432,7 @@ export const Winetricks = {
     runner: Runner,
     appName: string,
     args: string[],
-    event?: Electron.IpcMainInvokeEvent
+    returnOutput = false
```

Comment:
we didn't need the event for the messages to work so I removed that

I also added a new parameters to make the method return the output instead of sending messages, so we can capture the results of some commands